resources/public/api/conf/1.0/application.yaml

```diff
@@ -33,7 +33,7 @@ components:
   securitySchemes:
     User-Restricted:
       type: oauth2
-      description: HMRC supports OAuth 2.0 for authenticating [User-restricted](https://developer.qa.tax.service.gov.uk/api-documentation/docs/authorisation/user-restricted-endpoints) API requests
+      description: HMRC supports OAuth 2.0 for authenticating [User-restricted](https://developer.service.hmrc.gov.u/api-documentation/docs/authorisation/user-restricted-endpoints) API requests
```

Comment:
here the url should be `https://developer.service.hmrc.gov.uk/api-documentation/docs/authorisation/user-restricted-endpoints` instead of `https://developer.service.hmrc.gov.u/api-documentation/docs/authorisation/user-restricted-endpoints`. there is a `k` missing in the url.

Reply:
Thanks for spotting, done